test/CadlRanchProjects.Tests/model/inheritance/RecursiveTests.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using AutoRest.TestServer.Tests.Infrastructure;
+using NUnit.Framework;
+using _Type.Model.Inheritance.Recursive;
+using _Type.Model.Inheritance.Recursive.Models;
+using Azure.Core;
+using System.Linq;
+
+namespace CadlRanchProjects.Tests
+{
+    public class RecursiveTests : CadlRanchTestBase
+    {
+        [Test]
```

Comment:
Name this file as `model-inheritance-recursive.cs` and rename the class name to `ModelInheritanceRecursiveTests` to keep consistence.

Reply:
I want to consistent with other inheritance tests. For inheritance, it put in model->inheritance, and just use the project name as the test file name. So now, we will not update to ModelInheritanceRecursiveTests, And we will consistent all the cadl-ranch.test file and folder structure.